app/models/fact_check.rb

```diff
@@ -1,6 +1,8 @@
 class FactCheck < ApplicationRecord
   include Article
 
+  enum report_status: { unpublished: 0, paused: 1, published: 2 }
```

Comment:
Please change the order, @melsawy ... I think it's more clear if the order is:

```ruby
enum report_status: { unpublished: 0, published: 1, paused: 2 }
```

And the reason is because a report can only be paused if it was published before.